bentoml/deployment/serverless/aws_lambda.py

```diff
@@ -125,119 +130,156 @@ def generate_handler_py(bento_name, apis, output_path):
     with open(os.path.join(output_path, "handler.py"), "w") as f:
         f.write(AWS_HANDLER_PY_TEMPLATE_HEADER.format(class_name=bento_name))
         for api in apis:
-            api_content = AWS_FUNCTION_TEMPLATE.format(api_name=api['name'])
+            api_content = AWS_FUNCTION_TEMPLATE.format(api_name=api.name)
             f.write(api_content)
     return
 
 
 class AwsLambdaDeploymentOperator(DeploymentOperatorBase):
     def apply(self, deployment_pb, repo, prev_deployment=None):
```

Comment:
discussed offline, change the signature of deployment operator from `def apply(self, deployment_pb, repo, prev_deployment=None)` to `def apply(self, deployment_pb, yatai_service, prev_deployment=None)`